### ISSUE TITLE
Adding new forms/stances to SpellShapeShiftForm and importing to spell editor doesn't show new stances

### ISSUE THREAD

**Issue by Lambeg** [stoneharry/WoW-Spell-Editor#174]
**Spell editor version**
2.0.1

**Describe the bug**
Importing a new spellshapeshiftform.dbc doesn't make any new stances appear in the spell editor in stances.

**To Reproduce**
1. Edit spellshapeshiftform.dbc
2. Drop tables in spell editor
3. Import with spellshapeshiftform.dbc selected
4. Click a spell and go to flags
5. In Misc under stances. There will be no new stances added

**Expected behavior**
New stances should be added to list

**Screenshots**

**Additional context**

**Comment by stoneharry** [comment]
It is currently handled in the language file instead of being loaded from the DBC.

You can find it in Languages\enUS.xaml:
```xml
    <system:String x:Key="stances_strings">
        None |
        Cat |
        Tree |
        Travel |
        Aqua |
        Bear |
        Ambient |
        Ghoul |
        Dire Bear |
        Steves Ghoul |
        Tharonja Skeleton |
        Test of Strength |
        BLB Player |
        Shadow Dance |
        Creature Bear |
        Creature Cat |
        Ghost Wolf |
        Battle Stance |
        Defensive Stance |
        Berserker Stance |
        Test |
        Zombie |
        Metamorphosis |
        Undead |
        Master Angler |
        Flight (Epic) |
        Shadow |
        Flight (Normal) |
        Stealth |
        Moonkin |
        Spirit of Redemption
    </system:String>
```

You can change this as you want then restart the spell editor.

However, there is a known bug with stances, see:
- https://github.com/stoneharry/WoW-Spell-Editor/issues/160
- https://github.com/stoneharry/WoW-Spell-Editor/issues/174

**Comment by stoneharry** [comment]
I'm going to close this one because it answers your question and we can track the needed fix under #174.
